packages/controller/src/lib/setup.ts

```diff
@@ -3066,25 +3097,31 @@ async function checkSystemOffline(onlyCheck) {
 /**
  * Initialize plugins from io-pack and config json
  *
- * @param {object} config - parsed content of iobroker.json
- * returns {Promise<void>}
+ * @param config - parsed content of iobroker.json
  */
-function initializePlugins(config) {
+function initializePlugins(config: Record<string, any>): Promise<void> {
     const ioPackage = fs.readJsonSync(path.join(tools.getControllerDir(), 'io-package.json'));
     const packageJson = fs.readJsonSync(path.join(tools.getControllerDir(), 'package.json'));
     const hostname = tools.getHostName();
 
-    const pluginSettings = {
+    const pluginSettings: PluginHandlerSettings = {
         namespace: `system.host.${hostname}`,
         logNamespace: `host.${hostname}`,
         scope: 'controller',
         log: {
             // cli should be clean, only log warn/error
-            silly: _msg => {},
-            debug: _msg => {},
-            info: _msg => {},
-            warn: msg => console.log(msg),
-            error: msg => console.log(msg)
+            silly: (_msg: string) => {
```

Comment:
In fact we could hve a "defaultLogger" and use it in all 4 places ... then it would be eaier dto change it when needed? (Maybe then also addd a --debug parameter hwich enableds mroe logging? (not needingly this PR)